### PR TITLE
Update default value of `tidb_analyze_version` in v5.1 and v5.2

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update default value of `tidb_analyze_version` in v5.1.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/8411
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -279,7 +279,12 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Value options: `1` and `2`
 - Default value: `2`
```
Comment (qiancai) [inline system-variables.md:280]:
```suggestion
```

Comment (qiancai) [reply system-variables.md:280]: The Chinese version also needs to remove this line.

File: statistics.md [hunk 2]
```diff
@@ -5,7 +5,12 @@ summary: Learn how the statistics collect table-level and column-level informati
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1, v5.2, and v5.3, the default value of this variable is `2`, which serves as an experimental feature.
+TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. The default value of this variable in each version of v5.1.x is as follows:
+
+| Version | Default Value |
+| :- | :- |
+| v5.1.0 - v5.1.3 | `2`, serves as an experimental feature |
```
Comment (TomShawn) [inline statistics.md:12]:
```suggestion
| v5.1.0 - v5.1.3 | `2`, which serves as an experimental feature |
```

File: statistics.md [hunk 3]
```diff
@@ -5,7 +5,12 @@ summary: Learn how the statistics collect table-level and column-level informati
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1, v5.2, and v5.3, the default value of this variable is `2`, which serves as an experimental feature.
+TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. The default value of this variable in each version of v5.1.x is as follows:
+
+| Version | Default Value |
```
Comment (TomShawn) [inline statistics.md:10]:
```suggestion
| Version | Default value |
```

File: system-variables.md [hunk 4]
```diff
@@ -277,9 +277,13 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 - Scope: SESSION | GLOBAL
 - Value options: `1` and `2`
-- Default value: `2`
 - Controls how TiDB collects statistics.
-- In versions before v5.1.0, the default value of this variable is `1`. In v5.1.0, the default value of this variable is `2`, which serves as an experimental feature. For detailed introduction, see [Introduction to Statistics](/statistics.md).
+- The default value of `tidb_analyze_version` in each version of v5.1.x is as follows. For detailed introduction, see [Introduction to Statistics](/statistics.md).
```
Comment (TomShawn) [inline system-variables.md:281]:
```suggestion
- The default value of `tidb_analyze_version` in each version of v5.1.x is as follows. For detailed information, see [Introduction to Statistics](/statistics.md).
```

File: system-variables.md [hunk 5]
```diff
@@ -277,9 +277,13 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+
+| Version | Default Value |
+| :- | :- |
+| v5.1.0 - v5.1.3 | `2`, serves as an experimental feature |
```
Comment (TomShawn) [inline system-variables.md:285]:
```suggestion
| Version | Default value |
| :- | :- |
| v5.1.0 - v5.1.3 | `2`, which serves as an experimental feature |
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**